### PR TITLE
Add healthcheck reporting on build containers

### PR DESCRIPTION
**Pull request by jseppi**
Adds a "buildContainers" section to the healthcheck, and will display an error if there are too few containers found, or not all containers are started.

screen shot 2018-02-23 at 12 25 13 pm

### CODE REVIEW

**Comment by jseppi** [comment]
I need to fix-up some CodeClimate coverage/quality issues, but this is probably ready enough for review, @jmhooper.

File: src/cloud-foundry-api-client.js [hunk 1]
```diff
@@ -15,6 +19,29 @@ class CloudFoundryAPIClient {
...
+        const numBuildContainers = buildContainers.length;
+        const startedContainers = buildContainers.filter(bc => bc.state === STATE_STARTED);
+        const allStarted = startedContainers.length === expectedNumBuildContainers;
+
+        if (numBuildContainers < expectedNumBuildContainers) {
+          return { error: `Expected ${expectedNumBuildContainers} build containers but only ${numBuildContainers} found.` };
+        }
+
+        if (!allStarted) {
+          return { error: `Not all build containers are in the ${STATE_STARTED} state.` };
```
Comment (jmhooper) [inline src/cloud-foundry-api-client.js]: Any chance of a false positive here if we catch a build container that is in the middle of restarting?

Comment (jseppi) [reply src/cloud-foundry-api-client.js]: That's a good question. Maybe we should instead look for containers in the `STOPPED` state?

Comment (jmhooper) [reply src/cloud-foundry-api-client.js]: Yeah, that works. I think we'll know pretty soon if we've made a mistake and can revisit this then.

Comment (jseppi) [reply src/cloud-foundry-api-client.js]: After thinking about it some more, I actually think we should keep it as looking for `STARTED`. We'll want to know if something gets stuck in `PENDING` or some other state. After monitoring the prod containers for a bit, I haven't noticed them changing their state from `STARTED` even while builds are being staged.

But, if I did miss something, having this running in staging for the next week should let us catch it and adjust.

Comment (jmhooper) [reply src/cloud-foundry-api-client.js]: 👍

**Comment by jseppi** [comment]
ready for final review, @jmhooper.

**Comment by jseppi** [comment]
This is already deployed to staging, btw: https://federalist-builder-staging.fr.cloud.gov/healthcheck

**Pull request merged**